projects/js-packages/charts/src/components/bar-chart/bar-chart.tsx

```diff
@@ -26,6 +26,14 @@ export interface BarChartProps extends BaseChartProps< SeriesData[] > {
 	showZeroValues?: boolean;
 }
 
+const BAR_CHART_DEFAULTS = {
+	...SHARED_LEGEND_DEFAULTS,
+	legendShape: 'rect' as const,
+	withPatterns: false,
+	showZeroValues: false,
+	orientation: 'vertical' as const,
+} as const;
```

Comment:
I think we could add a little more type safety with 
```suggestion
const BAR_CHART_DEFAULTS: Partial< BarChartProps > = {
	...SHARED_LEGEND_DEFAULTS,
	legendShape: 'rect' as const,
	withPatterns: false,
	showZeroValues: false,
	orientation: 'vertical' as const,
};
```

Reply:
Marking this resolved as per @kangzj feedback further down to remove the default conts